client/question/services/code_evaluators/PythonCodeRunnerServiceSpec.js

```diff
@@ -64,6 +64,32 @@ describe('PythonCodeRunnerService', function() {
       $httpBackend.flush();
     });
 
+    it('processes a compile-time syntax error', function() {
+      var code = [
+        'def myFunction(arg):',
+        '    result = arg.rstrip()',
+        '    return result',
```

Comment:
I don't see a syntax error here

Reply:
It doesn't have or need one -- it mocks it out. I can add one, but it's not really in the scope of the function. Would you prefer a comment?

Reply:
I think you have two choices: (a) either make the code and error consistent so that it is a reasonable real-world input, or (b) make the code clearly say "mock code" and the error string clearly say "mock stderr" or something. (b) is probably better in order to avoid the reader thinking that the content actually matters.

Having pseudo-content that doesn't represent a real case just seems ... kind of confusing.